src/test/javascript/spec/component/iris/ui/chat-status-bar.component.spec.ts

```diff
@@ -0,0 +1,21 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { ChatStatusBarComponent } from 'app/iris/base-chatbot/chat-status-bar/chat-status-bar.component';
+
+describe('ChatStatusBarComponent', () => {
+    let component: ChatStatusBarComponent;
+    let fixture: ComponentFixture<ChatStatusBarComponent>;
+
+    beforeEach(async () => {
+        await TestBed.configureTestingModule({
+            imports: [ChatStatusBarComponent],
+        }).compileComponents();
+
+        fixture = TestBed.createComponent(ChatStatusBarComponent);
+        component = fixture.componentInstance;
+        fixture.detectChanges();
+    });
+
+    it('should create', () => {
+        expect(component).toBeTruthy();
+    });
+});
```

Comment:
What is the benefit of this test?

I think that we should either add actual tests or get rid of this file

Reply:
We will add actual tests, for now it's a barebone and has no purpose